src/Aquifer.API/Program.cs

```diff
@@ -55,6 +56,8 @@
 
 var app = builder.Build();
 
+StaticLoggerFactory.LoggerFactory = app.Services.GetRequiredService<ILoggerFactory>();
```

Comment:
Inspired by https://stackoverflow.com/a/50646048 so that I could have logging from extension methods without passing an `ILogger` parameter.